ui/src/app/components/workspace/workspace.component.html

```diff
@@ -15,13 +15,15 @@
     </div>
   </div>
   <div class="container">
-    <h2>Configure schema</h2>
     <div class="summary">
       Estimation for {{ srcDbName.toUpperCase() }} to for Spanner conversion:
       {{ conversionRatePercentages.good }}% of tables can be converted automatically,
       {{ conversionRatePercentages.ok }}% requires minimal conversion changes and
       {{ conversionRatePercentages.bad }}% requires high complexity conversion changes.
     </div>
+    <div class="summary">
```

Comment:
Why a separate div for this ? Also better to rephrase as: To make schema changes go to Spanner Draft pane.

Reply:
Add a link to Spanner draft pane

Reply:
Updated in the ux-changes-v2 branch.